torch_geometric/nn/glob/__init__.py

```diff
@@ -18,8 +16,14 @@
 
 from torch_geometric.deprecation import deprecated  # noqa
 from torch_geometric.nn.aggr import Set2Set  # noqa
+from torch_geometric.nn.glob import sort  # noqa
 
 Set2Set = deprecated(
     details="use 'nn.aggr.Set2Set' instead",
     func_name='nn.glob.Set2Set',
 )(Set2Set)
+
+global_sort_pool = deprecated(
+    details="use 'nn.aggr.GlobalSortAggr' instead",
+    func_name='nn.glob.global_sort_pool',
+)(sort.global_sort_pool)
```

Comment:
cannot use the new class here as its not a replacement (new class doesn't take `k` in the function call)

Reply:
we could make it a kwarg if we want it to work exactly the same way

Reply:
I think you can just define it here:
```python
@deprecated(...)
def global_sort_pool(...):
    module = GlobalSortAggr(...)
    return module(...)
```

Reply:
yeah okay this is a better solution, thanks!